tests/conftest.py

```diff
@@ -392,8 +392,8 @@ def mock_rss(input_dir: str = None,
                        dir=input_dir,
                        random_seed=random_seed)
     job3 = DFTStaticLabelling(e0_spin=e0_spin,
-                              isolated_atom=isolated_atom,
-                              dimer=dimer,
+                              include_isolated_atom=isolated_atom,
```

Comment:
I was wondering if we still need the `mock_do_rss_iterations_multi_jobs` and `mock_do_rss_iterations` functions, as we have a complete RSS workflow test in place?